recbole_gnn/data/dataset.py

```diff
@@ -101,6 +101,54 @@ def build(self):
             dataset.session_graph_construction()
         return datasets
 
+class MultiBehaviorDataset(SessionGraphDataset):
+
+    def session_graph_construction(self):
+        self.logger.info('Constructing Multi-behavior session graphs.')
+        self.item_behavior_list_field = self.config['ITEM_BEHAVIOR_LIST_FIELD']
+        item_seq = self.inter_feat[self.item_id_list_field]
+        item_seq_len = self.inter_feat[self.item_list_length_field]
+        if self.item_behavior_list_field == None:
+            # To be compatible with existing datasets
+            item_behavior_seq = torch.tensor([0] * len(item_seq))
+            self.field2id_token['behavior_id'] = {0:'interaction'}
```

Comment:
```suggestion
            self.behavior_id_field = config['Behavior_ID_FIELD']
            self.field2id_token[self.behavior_id_field] = {0: 'interaction'}
```

Maybe the field name of behavior ID can also be customized? 
```yaml
Behavior_ID_FIELD: behavior_id
```
or fixed as `behavior_id` is also ok. Just a suggestion.